pandas/_libs/tslibs/timedeltas.pyi

```diff
@@ -146,3 +146,7 @@ class Timedelta(timedelta):
     def __hash__(self) -> int: ...
     def isoformat(self) -> str: ...
     def to_numpy(self) -> np.timedelta64: ...
+    @property
+    def freq(self) -> None: ...
+    @property
+    def is_populated(self) -> bool: ...
```

Comment:
Could also remove the type annotation to not encourage deprecated behavior.

For all the overloads that depend on keyword-only arguments, the type annotations already suggest that positional arguments are no longer supported (even though the implementation is still in the deprecation phase).

Reply:
yeah we don't really have a policy here for deprecated functions. i could go either way.